### ISSUE TITLE
Add multiple game support?

### ISSUE THREAD

**Issue by naclcaleb** [OOPS-Studio/Battleship-ONLINE-Server#1]
This is a cool project, but if more people start using it, you're going to need to make it so more than one person can be playing at once. That being said, it would introduce a few new challenges, but it might still be worth it for you.

(BTW, I know this says "issue", but these are also used for feature requests and things like that, in case you didn't know)

**Comment by naclcaleb** [comment]
And this is just a suggestion; do with it what you will.

**Comment by OOPS-Studio** [comment]
Yes! That is actually what I'm working on right now! ;) It won't be too hard! ;) (Just break player connectons into arrays in an array and manage them that way! ;)) Thanks for suggesting this anyway! ;)

**Comment by OOPS-Studio** [comment]
Also, if you check out the "Projects" section, you can see what I am currently/planning on working on! ;)

**Comment by naclcaleb** [comment]
Ah, I see. Thanks.

**Comment by OOPS-Studio** [comment]
NP! ;)
